### PR TITLE
storage: Allow mounting all NFS directories

### PR DESCRIPTION
**Pull request by mvollmer**
and not only those that are in the drop down list.  The list is not
always accurate and the user might have a better idea of what will
work.

### CODE REVIEW

File: pkg/storaged/dialog.jsx [hunk 1]
```diff
@@ -537,6 +537,8 @@ const TypeAheadSelectElement = ({ options, change }) => {
     return (
         <TypeAheadSelect
             variant={SelectVariant.typeahead}
+            isCreatable
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply pkg/storaged/dialog.jsx:540]: Int this case yes let's keep the isCreatetable property. Sorry for the late answer, missed the notification.

File: test/verify/check-storage-nfs [hunk 2]
```diff
@@ -170,6 +182,35 @@ class TestStorageNfs(StorageCase):
...
+
+        m.execute("mkdir /home/foo /home/bar")
+        m.write("/etc/exports", "/home/foo 127.0.0.0/24(rw)\n/home/bar 127.0.0.0/24(rw)\n")
+        m.execute("systemctl restart nfs-server")
+
+        # Break showmount.  Cockpit uses showmount to list exported
+        # directories, but that relies on a properly configured
+        # firewall etc.  Even if showmount doesn't work, Cockpit
+        # should allow people to mount arbitrary directories.
+        m.execute("chmod a-x /usr/sbin/showmount")
```
Comment (KKoukiou) [inline test/verify/check-storage-nfs:199]: This needs to be cleaned up in teardown, so an addCleanup  is necessary here.

Comment (mvollmer) [reply test/verify/check-storage-nfs:199]: This is not a `@nondestructive` test, though.

**Review by KKoukiou — APPROVED** [review]

File: test/verify/check-storage-nfs (hunk 2 above)
Comment (KKoukiou) [reply test/verify/check-storage-nfs:199]: Aha! Did not realize, thanks!

**Pull request merged**